pybv/io.py

```diff
@@ -74,6 +74,10 @@ def write_brainvision(data, sfreq, ch_names, fname_base, folder_out,
         ('u' stands for microseconds). Note that setting a measurement date
         implies that one additional event is created in the .vmrk file. To
         prevent this, set this parameter to None (default).
+    unit : str
+        The unit of the exported data. This can be one of 'V', 'mV', 'uV',
+        'µV', 'nV' or 'auto'. If 'auto', a suitable unit based on the selected
+        resolution is chosen automatically.
```

Comment:
maybe mention that µV and uV are intended as synonyms with uV being the ASCII representation of µV

Reply:
OK!